static/js/apps/eval_rig/query_section.tsx

```diff
@@ -37,12 +38,10 @@ export function QuerySection(props: QuerySectionProps): JSX.Element {
   const [answer, setAnswer] = useState<string>("");
 
   const loadAnswer = (doc: GoogleSpreadsheet, rowIndex: number) => {
-    doc.loadInfo().then(() => {
-      const qaSheet = doc.sheetsByTitle["query_and_answer"];
-      const address = `D${rowIndex + 1}`;
-      qaSheet.loadCells(address).then(() => {
-        setAnswer(qaSheet.getCellByA1(address).value as string);
-      });
+    const qaSheet = doc.sheetsByTitle[QASheet];
+    const address = `D${rowIndex + 1}`;
+    qaSheet.loadCells(address).then(() => {
+      setAnswer(qaSheet.getCellByA1(address).value as string);
```

Comment:
Any reason why this doesn't use `ANSWER_COL`?

Reply:
it should! Updated